packages/flutter_tools/test/general.shard/artifacts_test.dart

```diff
@@ -84,7 +84,7 @@ void main() {
 
       fileSystem
           .directory(xcframeworkPath)
-          .childDirectory('ios-x86_64-simulator')
+          .childDirectory('ios-arm64_x86_64-simulator')
```

Comment:
The changes to this file aren't strictly related to the codesign issue, but may as well change them together.